app/v1/models/response/retrieveTransaction/TransactionItem.scala

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v1.models.response.retrieveTransaction
+
+import play.api.libs.json.{JsPath, Json, OWrites, Reads}
+import play.api.libs.functional.syntax._
+import v1.models.domain.DesTaxYear
+
+case class TransactionItem(taxYear: Option[String],
+                           id: Option[String],
```

Comment:
This field needs to dynamically read in either a paymentId (in the case that paymentLot and paymentLotItem fields are present), or a chargeId (in the case that only a documentId field is available)